lib/how_is/report/html.rb

```diff
@@ -19,7 +19,7 @@ def header(content)
     end
 
     def link(content, url)
-      %[<a href="#{url}">#{content}</a>]
+      %(<a href="#{url}">#{content}</a>)
```

Comment:
as @houndci-bot pointed out, I actually had changed the configuration to use []. I'm not sure how/why RuboCop would've been giving a warning about this. O.o